osu.Game.Tournament/Screens/StablePathSelectScreen.cs

```diff
@@ -42,11 +38,6 @@ private void load(Storage storage, OsuColour colours)
         {
             var initialPath = new DirectoryInfo(storage.GetFullPath(string.Empty)).Parent?.FullName;
 
-            if (!string.IsNullOrEmpty(stableInfo.StablePath))
```

Comment:
This is a little nitpick, but right now the DirectorySelector always defaults to the lazer data directory. In many cases users have their stable tournament install located next to their normal installation in the form of: 

``` 
<ParentDirectory>/osu!
<ParentDirectory>/osu! Tourney
```

Having it default to the parent directory of the default install (if detected) would make it more easier for the user to the get to the correct installation.

Reply:
I was going to keep it but forgot about it along the way. The previous behaviour should be back now with my latest commit.